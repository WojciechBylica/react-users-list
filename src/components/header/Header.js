import { Nav } from '../index';
import './style.scss';
import image from './logo.png';
import { NavLink } from 'react-router-dom';

const Header = () => (
  <header className="header">
    <div className="header__box">
      <NavLink className="navigation__link navigation__link--header" to="/">
        <h1 className="header__title">
          <img width={50} height={50} alt="logo" src={image} />
          Users List
        </h1>
      </NavLink>
      <Nav />
    </div>
  </header>
);

export default Header;
