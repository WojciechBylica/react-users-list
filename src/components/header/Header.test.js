import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Header from '.';

it('should render ok', () => {
  const tree = renderer.create(
    <BrowserRouter>
      <Header />
    </BrowserRouter>
  );
  expect(tree).toMatchSnapshot();
});
