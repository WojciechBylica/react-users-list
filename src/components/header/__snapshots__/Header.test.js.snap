// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ok 1`] = `
<header
  className="header"
>
  <div
    className="header__box"
  >
    <h1
      className="header__title"
    >
      Users List
    </h1>
    <nav>
      <a
        className="navigation__link"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </nav>
  </div>
</header>
`;
